From: Mathieu Malaterre <malat@debian.org>
Date: Fri, 28 Jan 2022 12:00:32 +0100
Subject: lintian hardening-no-bindnow

Forwarded: not-needed
Last-Update: 2022-01-28
---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 53203e2..bf564de 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,6 +6,9 @@ project(jxrlib C)
 # Need shared libs for ABI
 set(BUILD_SHARED_LIBS ON)
 
+# PIE for executables:
+set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
+
 # helper macro to preserve original Makefile convention
 macro(JXR_MAKE_OBJ SET_NAME)
   foreach(src ${SRC_${SET_NAME}})
