From: Mathieu Malaterre <malat@debian.org>
Date: Fri, 6 Nov 2015 21:57:27 +0100
Subject: Fix typos and remove some warnings

---
 jxrencoderdecoder/JxrEncApp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/jxrencoderdecoder/JxrEncApp.c b/jxrencoderdecoder/JxrEncApp.c
index aa9de53..63cbaf8 100644
--- a/jxrencoderdecoder/JxrEncApp.c
+++ b/jxrencoderdecoder/JxrEncApp.c
@@ -615,7 +615,7 @@ main(int argc, char* argv[])
 
     //================================
     Call(PKCreateCodecFactory(&pCodecFactory, WMP_SDK_VERSION));
-    Call(pCodecFactory->CreateCodec(&IID_PKImageWmpEncode, &pEncoder));
+    Call(pCodecFactory->CreateCodec(&IID_PKImageWmpEncode, (void**)&pEncoder));
 
     //----------------------------------------------------------------
     Call(PKCreateTestFactory(&pTestFactory, WMP_SDK_VERSION));
